ui/Modal/ManageRemotes.svelte

```diff
@@ -0,0 +1,145 @@
+<script lang="ts">
+  import { Button, Input } from "../DesignSystem/Primitive";
+  import { Illustration, List, Remote } from "../DesignSystem/Component";
+
+  import Peer from "./ManageRemotes/Peer.svelte";
+  import PeerFollowRequest from "./ManageRemotes/PeerFollowRequest.svelte";
+
+  import type { PeerId } from "../src/identity";
+  import type { Urn } from "../src/urn";
+
+  import { Variant as IllustrationVariant } from "../src/illustration";
+  import {
+    addPeer,
+    pendingPeers,
+    peerSelection,
+    peerValidation,
+    project as store,
+    removePeer,
+  } from "../src/project";
+
+  let newPeer: PeerId;
+
+  $: if (newPeer === "") {
+    peerValidation.reset();
+  }
+
+  const submitPeer = async (projectUrn: Urn) => {
+    if (await addPeer(projectUrn, newPeer)) {
+      newPeer = "";
```

Comment:
Just to check, does the `peerValidation.reset()` above cover this case?

Reply:
Correct, it does.